backend/api/user.go

```diff
@@ -3,12 +3,26 @@ package api
 import (
 	"github.com/gin-gonic/gin"
 	"net/http"
-	//"fmt"
+
 	"backend/internal/pkg"
 	"backend/internal/model"
 	"strconv"
 )
 
+// Register godoc
+// @Summary      register a user
+// @Description  add a user to db
+// @Tags         user
+// @Accept       json
+// @Produce      json
+// @Param   	 username formData string true "user's name"
+// @Param   	 password formData string true "password"
+// @Param   	 overview formData string false "overview"
+// @Param   	 organization formData string false "organization"
+// @Success      200  {object}  model.Response "success"
+// @Failure      400  {object}  model.Response "username exists | password & confirmPassword not consistent"
+// @Failure 	 500  {object}  model.Response "db error"
+// @Router       /user [post]
 func Register(c *gin.Context) {
```

Comment:
400 bad request with user conflict and password.
Maybe password inconsistent should be checked at frontend.

Reply:
@rain72510 
Only one password needed to be sent to the backend.

Reply:
fixed